reana_client/api/client.py

```diff
@@ -344,9 +343,10 @@ def download_file(workflow_id, file_name, access_token):
             return http_response.content
         else:
             raise Exception(
-                "Expected status code 200 but replied with "
-                "{status_code}".format(
-                    status_code=http_response.status_code))
+                "Error {status_code} {reason} {message}".format(
+                    status_code=http_response.status_code,
+                    reason=http_response.reason,
+                    message=http_response.json().get('message')))
```

Comment:
Thanks, it looks better, but sometimes I see the message gets long:

A. this is nice:

```console
$ reana-client download wrong.txt -w root6-roofit-cwl
File wrong.txt could not be downloaded: Error 404 Not Found wrong.txt does not exist.
```

B. this is less nice:

```console
$ reana-client download wrong.txt -w notexisting
File wrong.txt could not be downloaded: Error 404 Not Found REANA_WORKON is set to notexisting, but that workflow does not exist. Please set your REANA_WORKON environment variable appropriately.
```

I can ticketise for later...